PyRoute/StatCalculation.py

```diff
@@ -236,6 +236,10 @@ def add_pop_to_sophont(self, stats, star):
             soph_code = sophont[0:4]
             soph_pct = sophont[4:]
 
+            if 1 < len(soph_pct) and '(' == soph_code[0]:
```

Comment:
I'm confused about this change. It looks like the TradeCode processor for the home worlds leaves the parans as part of the name. (e.g. "(Slye)4") as the code. Largely the purpose of the `StatsCalculaion` is to output files for the wiki. Which doesn't know about the new standard here. This should trim the parans from the name so the "soph_code" comes out as "Slye" and not "(Slye)"

Reply:
(Syle)4 now unpacks to Syle4, and likewise for other pop levels

Reply:
If the change is correct, this is now dead code?

Reply:
No, it's still needed to unpack from TradeCodes - such as your (Syle)4 example.